SPECS/linstor-client.spec

```diff
@@ -0,0 +1,53 @@
+Summary: DRBD distributed resource management utility
+Name:    linstor-client
+Version: 1.19.0
+Release: 1%{?dist}
+License: GPL-3.0-or-later
+URL:     https://linbit.com/linstor/
+
+BuildArch: noarch
+BuildRequires: python3-setuptools
+
+Requires: python-linstor >= 1.19.0
+
+Source0: https://pkg.linbit.com//downloads/linstor/%{name}-%{version}.tar.gz
+
+%define python "python3"
+
+%description
+This client program communicates to controller node which manages the resources
+
+%prep
+%autosetup -p1
+
+%build
+PYTHON=%{python} %{python} ./setup.py build
+
+%install
+PYTHON=%{python} %{python} ./setup.py install --single-version-externally-managed -O1 --root=$RPM_BUILD_ROOT --record=INSTALLED_FILES
+
+%files
+/usr/bin/linstor
+/etc/bash_completion.d/linstor
+/usr/lib/python3.6/site-packages/linstor_client_main.py
+/usr/lib/python3.6/site-packages/__pycache__/linstor_client_main.cpython-36.pyc
+/usr/lib/python3.6/site-packages/__pycache__/linstor_client_main.cpython-36.opt-1.pyc
+/usr/lib/python3.6/site-packages/linstor_client/*.py
+/usr/lib/python3.6/site-packages/linstor_client/__pycache__/*.pyc
+/usr/lib/python3.6/site-packages/linstor_client/argparse/*.py
+/usr/lib/python3.6/site-packages/linstor_client/argparse/__pycache__/*.pyc
+/usr/lib/python3.6/site-packages/linstor_client/argcomplete/*.py
+/usr/lib/python3.6/site-packages/linstor_client/argcomplete/__pycache__/*.pyc
+/usr/lib/python3.6/site-packages/linstor_client/commands/*.py
+/usr/lib/python3.6/site-packages/linstor_client/commands/__pycache__/*.pyc
+/usr/lib/python3.6/site-packages/linstor_client-1.19.0-py3.6.egg-info/SOURCES.txt
+/usr/lib/python3.6/site-packages/linstor_client-1.19.0-py3.6.egg-info/dependency_links.txt
+/usr/lib/python3.6/site-packages/linstor_client-1.19.0-py3.6.egg-info/PKG-INFO
+/usr/lib/python3.6/site-packages/linstor_client-1.19.0-py3.6.egg-info/requires.txt
+/usr/lib/python3.6/site-packages/linstor_client-1.19.0-py3.6.egg-info/top_level.txt
+
+%license COPYING
```

Comment:
I think we could add the readme: `%doc README.md`

Reply:
Added in all repos/branches